pywhat/identifier.py

```diff
@@ -3,9 +3,8 @@
 from typing import Callable, Optional
 
 from pywhat.filter import Distribution, Filter
-from pywhat.helper import AvailableTags, Keys

```

Comment:
why?

Reply:
It isn't used in the file and that's why I removed it.

Reply:
oki